lib/zendesk_apps_support/assets/installed.js.erb

```diff
@@ -14,4 +14,8 @@
   <% end %>
 }());
 
+<% unless rollbar_zaf_access_token.empty? %>
```

Comment:
if this code goes live server-side before the variable is available, or if running in zat